arcgis-ios-sdk-samples/Utility Network/Configure subnetwork trace/README.metadata.json

```diff
@@ -0,0 +1,71 @@
+{
+    "category": "Utility Network",
+    "description": "Get a server-defined trace configuration for a given tier and modify its traversability scope, add new condition barriers and control what is included in the subnetwork trace result.",
+    "ignore": false,
+    "images": [
+        "configure-subnetwork-trace.png"
+    ],
+    "keywords": [
+        "category comparison",
+        "condition barriers",
+        "network analysis",
+        "network attribute comparison",
+        "subnetwork trace",
+        "trace configuration",
+        "traversability",
+        "utility network",
+        "validate consistency",
+        "AGSCodedValueDomain",
+        "AGSUtilityAssetType",
+        "AGSUtilityAttributeComparisonOperator",
+        "AGSUtilityCategory",
+        "AGSUtilityCategoryComparison",
+        "AGSUtilityCategoryComparisonOperator",
+        "AGSUtilityDomainNetwork",
+        "AGSUtilityElement",
+        "AGSUtilityElementTraceResult",
+        "AGSUtilityNetwork",
+        "AGSUtilityNetworkAttribute",
+        "AGSUtilityNetworkAttributeComparison",
+        "AGSUtilityNetworkDefinition",
+        "AGSUtilityTerminal",
+        "AGSUtilityTier",
+        "AGSUtilityTraceAndCondition",
+        "AGSUtilityTraceConfiguration",
+        "AGSUtilityTraceOrCondition",
+        "AGSUtilityTraceParameters",
+        "AGSUtilityTraceResult",
+        "AGSUtilityTraceType",
+        "AGSUtilityTraversability"
+    ],
+    "redirect_from": [],
+    "relevant_apis": [
+        "AGSCodedValueDomain",
+        "AGSUtilityAssetType",
+        "AGSUtilityAttributeComparisonOperator",
+        "AGSUtilityCategory",
+        "AGSUtilityCategoryComparison",
+        "AGSUtilityCategoryComparisonOperator",
+        "AGSUtilityDomainNetwork",
+        "AGSUtilityElement",
+        "AGSUtilityElementTraceResult",
+        "AGSUtilityNetwork",
+        "AGSUtilityNetworkAttribute",
+        "AGSUtilityNetworkAttributeComparison",
+        "AGSUtilityNetworkDefinition",
+        "AGSUtilityTerminal",
+        "AGSUtilityTier",
+        "AGSUtilityTraceAndCondition",
+        "AGSUtilityTraceConfiguration",
+        "AGSUtilityTraceOrCondition",
+        "AGSUtilityTraceParameters",
+        "AGSUtilityTraceResult",
+        "AGSUtilityTraceType",
+        "AGSUtilityTraversability"
+    ],
+    "snippets": [
+        "ConfigureSubnetworkTraceViewController.swift",
+        "ConfigureSubnetworkTraceOptionsViewController"
```

Comment:
Still missing the swift extension 😅 
In the future I'll make a drag and drop tool to generate metadata from folder, to replace our current script.